CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing
+
+Thanks for helping to make gql awesome!
+
+We welcome all kinds of contributions:
+
+- Bug fixes
+- Documentation improvements
+- New features
+- Refactoring & tidying
+
+
+## Getting started
+
+If you have a specific contribution in mind, be sure to check the [issues](https://github.com/graphql-python/gql/issues) and [pull requests](https://github.com/graphql-python/gql/pulls) in progress - someone could already be working on something similar and you can help out.
+
+
+## Project setup
+
+After cloning this repo, ensure dependencies are installed by running:
+
+```sh
+make dev-setup
```

Comment:
Can't we just put the actual command `pip install -e .[test]` here? Hiding this step in a makefile makes intransparent what is happening in this step. You also don't want to frustrate people using Windows which does not have a make command. Also, you should recommend using a virtual env. Otherwise the dependencies will be all installed into the global one.

Reply:
I followed the `graphene-django` documentation as base for gql docs, including the Makefile but you're right, it is better to provide the actual command instead of using a alias on the Makefile.

For Windows users, I found it is better to use either the conda development steps or we should rely on some `Make.bat` as equivalent.

Reply:
Right, but not all people on Windows are using Conda. It should also not be emphasized too much because otherwise the Windows user might think they *must* use Conda to develop gql.

Reply:
I will make the clarifications at the night so users can have multiple choices to setup a development environment.